src/Compilers/Core/Portable/Xml/XmlCharType.cs

```diff
@@ -94,7 +95,7 @@ internal static class XmlCharType
 
 #if XMLCHARTYPE_USE_LITERAL_TABLE
         // this tables can be re-generated by the Dump function below.
-        private static readonly byte[] s_charPropertiesIndex =
+        private static readonly byte[] s_charPropertiesIndex = new byte[]
```

Comment:
Is this a style-only change?

Reply:
Oops. I was originally going to change that field to a `ReadOnlySpan<byte>` property, but couldn't because that project doesn't reference core libraries that include span types. I'll revert that change.